docs/sources/operations/scalability.md

```diff
@@ -12,3 +12,11 @@ partitioned by role (ingester, distributor, querier) rather than a single Loki
 process. Grafana Labs' [production setup](https://github.com/grafana/loki/blob/master/production/ksonnet/loki)
 contains `.libsonnet` files that demonstrates configuring separate components
 and scaling for resource usage.
+
+## Separate Query Scheduler
+
+The Query frontend has an in-memory queue that can be moved out into a separate process similar to the [Cortex Query Scheduler](https://cortexmetrics.io/docs/operations/scaling-query-frontend/#query-scheduler). This allows running multiple query frontends.
+
+In order to run with the Query Scheduler, the frontend needs to be passed the scheduler's address via `-frontend.scheduler-address` and the querier processes needs to be started with `-querier.scheduler-address` set to the same address. It is not valid to set `-querier.frontend-address` and `-querier.frontend-address`. Both options can also be set via the [configuration file](../configuration).
```

Comment:
Please revise the second sentence.  I don't know what it intends to say, but you have the same `-querier.frontend-address`  twice in the sentence.  If you're trying to specify that you cannot set both items in the configuration, then I'd prefer to see wording similar to:
Specify only one of `first-one` and `second-one` when invoking the <fill in here>; it is not valid to specify both.

Reply:
Thanks. This is a typo. IT's not valid to specify `querier.frontend-address` and `querier.scheduler-address`, I'll clean this up.